distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -183,6 +183,12 @@ org.codice.ddf.platform.util.http.initialRetryInterval=10
 javax.xml.parsers.DocumentBuilderFactory=org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
 javax.xml.transform.TransformerFactory=org.apache.xalan.processor.TransformerFactoryImpl
 
+#
```

Comment:
❓Wonder how much we should "advertise" this system property. For instance, we don't mention `catalog.maxPageSize` anywhere. How likely is it that an admin would need to change the default value? @brendan-hofmann, @beyelerb, @pklinef, thoughts?

Reply:
I don't have strong feelings either way.  Though I do wonder where should we start tracking all of the system properties we support but do not expose?

Reply:
That did cross my mind. Maybe have some kind of naming convention for those? Annotation maybe?

Reply:
I have removed that from the properties